### PR TITLE
Story/dfc 8194 breadcrumb contact us

### PR DESCRIPTION
**Pull request by trevorkapswarah**

### CODE REVIEW

File: DFC.Digital/DFC.Digital.Web.Sitefinity.Core/SitefinityExtensions/WebAppContext.cs [hunk 1]
```diff
@@ -47,6 +47,8 @@ public bool IsContentPreviewMode
 
         public bool IsSearchResultsPage => HttpContext.Current.Request.Url.ToString().ToLower().Contains("/search-results");
 
+        public bool IsContactUsPage => HttpContext.Current.Request.Url.ToString().ToLower().Contains("/contact-us");
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.Core/SitefinityExtensions/WebAppContext.cs]: Where is this used?

Comment (trevorkapswarah) [reply DFC.Digital/DFC.Digital.Web.Sitefinity.Core/SitefinityExtensions/WebAppContext.cs]: removed

**Comment by trevorkapswarah** [comment]
@Muthuramana  ready for review

**Comment by trevorkapswarah** [comment]
@Muthuramana Please approve, you had approved but after merge it needs you to approve again

**Pull request merged**